ts/components/conversation/message/message-content/MessageContent.tsx

```diff
@@ -156,7 +139,6 @@ export const MessageContent = (props: Props) => {
     <StyledMessageContent
       className={classNames('module-message__container', `module-message__container--${direction}`)}
       role="button"
-      onClick={onClickOnMessageInnerContainer}
       title={toolTipTitle}
```

Comment:
What does removing this fix?

Reply:
well, that was the callback. Can you see why I've removed it?

```ts
// TODO not too sure what is this doing? It is not preventDefault()
// or stopPropagation() so I think this is never cancelling a click event?
function onClickOnMessageInnerContainer(event: MouseEvent<HTMLDivElement>) {
  const selection = window.getSelection();
  // Text is being selected
  if (selection && selection.type === 'Range') {
    return;
  }

  // User clicked on message body
  const target = event.target as HTMLDivElement;
  if (target.className === 'text-selectable' || window.contextMenuShown) {
    return;
  }
}
```

Reply:
Ah okay cool I got it. Resolving